examples/DisplayLamps/DisplayLamps.ino

```diff
@@ -10,69 +10,38 @@
  *
  */
 
-#include <TimerOne.h>
 #include "Dimmer.h"
 
-Dimmer dimmer;
-
-#define p1  10
-#define p2  40
-
-byte pot[6][3] = {  {p2,  0, 0 }, 
-                    {p1, p1, 0 },
-                    {0 , p2, 0 }, 
-                    {0 , p1, p1}, 
-                    {0 ,  0, p2},
-                    {p1,  0, p1}
-                  };
+//Set 3 dimerable lights at pins 3,5 and 6
```

Comment:
Let's use a space after `//`. Maybe we should use some kind of auto-formatting tool or IDE to do that for us. Even the Arduino IDE auto-formatting might do the trick.